config/initializers/secure_headers.rb

```diff
@@ -0,0 +1,23 @@
+SecureHeaders::Configuration.default do |config|
+
+  # Unblock PDF downloading for student report
+  config.x_download_options = nil
```

Comment:
We could also set this in the controller action so that it's only allowed for that the actual download request: https://github.com/twitter/secureheaders/blob/master/docs/per_action_configuration.md

Reply:
Took a closer look at this with @kevinrobinson and we realized that `x_download_options` isn't available as a per-controller configuration in their DSL. 😞